packages/marketplace/src/actions/settings/__tests__/settings.ts

```diff
@@ -0,0 +1,15 @@
+import { settingLoadingVisibility, changePassword } from '../settings'
+import ActionTypes from '@/constants/action-types'
+
+describe('settings actions', () => {
+  it('should create a settingShowLoading action', () => {
```

Comment:
so does it settingShowLoading or settingLoadingVisibility ?
But I think no need to setSetingLoadingAction, you can add loading to fetch action instead of add new action to setLoading